src/pocketmine/utils/TextFormat.php

```diff
@@ -387,6 +392,10 @@ public static function toHTML($string) : string{
 					$newString .= "<span style=color:#FFF>";
 					++$tokens;
 					break;
+				case TextFormat::DARK_YELLOW:
+					$newString .= "<span style=color:#DDD605>";
```

Comment:
Is there no 3-digit code that can represent this?

Reply:
No there is not, using an HTML colour picker you can see that #DDD605 is the correct colour code for it. This was also [confirmed by Damian](https://discordapp.com/channels/373199722573201408/373199722573201410/663031808593494026) when discussing the original pull request on discord.